data/languages.json

```diff
@@ -55,9 +55,11 @@
   "bew": {},
   "bez": {"nativeName": "Hibena"},
   "bfd": {},
+  "bft": {"nativeName": "بلتی"},
```

Comment:
This file was generated from [cldr-localenames-full](https://www.npmjs.com/package/cldr-localenames-full/) v37. Maybe upgrading [this dependency](https://github.com/openstreetmap/iD/blob/a2cacaaf2456de4c8443a7b5dab0ac3393be5deb/package.json#L81) to v41 would bring in some more language names so we wouldn’t have to maintain them manually in this repository.